apps/src/code-studio/components/progress/UnitOverview.jsx

```diff
@@ -59,6 +61,7 @@ class UnitOverview extends React.Component {
     publishedState: PropTypes.oneOf(Object.values(PublishedState)),
 
     // redux provided
+    unitTitle: PropTypes.string.isRequired,
```

Comment:
I'm seeing this component is being imported in our `apps/src/sites/studio/pages/scripts/show.js` file. How will this be populated from there? Could we instead use the `courseTitle` prop that already exists?

Reply:
Ah, I'm guessing course name and unit name might be different –– still not sure where unitTitle is coming from, and if it should be passed in explicitly from the `show.js` page

Reply:
Yes good point! Initially I was worried about courseTitle vs. unitTitle vs. scriptName etc. and I thought this was the most appropriate but in looking back I think scriptName provides the most accurate description for what the event is looking for (which is a prop already passed in) so I'll just simplify this. Thank you!